server/metrics/metrics.go

```diff
@@ -94,6 +100,25 @@ var (
 		},
 		[]string{"name"},
 	)
+
+	// CRIOContainersOOMTotal collects container out of memory (oom) metrics for every container and sandboxes.
+	CRIOContainersOOMTotal = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Subsystem: subsystem,
+			Name:      CRIOContainersOOMTotalKey,
+			Help:      "Amount of containers killed because they ran out of memory (OOM)",
+		},
+	)
+
+	// CRIOContainersOOM collects container out of memory (oom) metrics per container and sandbox name.
+	CRIOContainersOOM = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Subsystem: subsystem,
+			Name:      CRIOContainersOOMKey,
+			Help:      "Amount of containers killed because they ran out of memory (OOM) by their name",
```

Comment:
Is this really the "Amount" or the "The names"?

Reply:
I've just backported #4690 here, but I already filled #5426 to rework the metrics in general.